subworkflows/local/transcript_identification.nf

```diff
@@ -8,7 +8,9 @@ include { PINTS_CALLER } from '../../modules/nf-core/pints/caller/main'
 include { CAT_CAT } from '../../modules/nf-core/cat/cat/main'
 include { BEDTOOLS_MERGE } from '../../modules/nf-core/bedtools/merge/main'
 include { BEDTOOLS_SORT } from '../../modules/nf-core/bedtools/sort/main'
-include { BEDTOOLS_INTERSECT as BEDTOOLS_INTERSECT_FILTER } from '../../modules/nf-core/bedtools/intersect/main'
+include {
+    BEDTOOLS_INTERSECT as BEDTOOLS_INTERSECT_FILTER
+    BEDTOOLS_INTERSECT } from '../../modules/nf-core/bedtools/intersect/main'
```

Comment:
Why not 2 include statements? I feel it's easier to read and understand quickly.